zwave_js_server/model/controller/__init__.py

```diff
@@ -574,17 +575,18 @@ async def async_get_associations(
         for key, association_addresses in data["associations"].items():
             associations[int(key)] = [
                 AssociationAddress(
+                    self,
                     node_id=association_address["nodeId"],
                     endpoint=association_address.get("endpoint"),
                 )
                 for association_address in association_addresses
             ]
         return associations
 
-    async def async_is_association_allowed(
+    async def async_check_association(
```

Comment:
If we remove the old method I don't think it makes sense to support the old schema (36).